### DISCUSSION TITLE
v3.37.7

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#2340] [Announcements]
### Maintenance

* Update plugin commits to most recent commits

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.37.6...v3.37.7

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.37.7'>v3.37.7</a>.</em>

**Comment by chrisfair** [comment]
I have a question about how to do something and I simply have no idea where to ask.   I have a C++ document I am editing that has mixed encodings and it is not supported by one of the plugins, otherwise it does not seem to cause any issues, is there a way to turn off the notification for that file type for warnings?   Otherwise, wow, awesome config you guys have set up.

![image](https://github.com/AstroNvim/AstroNvim/assets/4054917/7c5846b1-b7c4-4772-b7de-f8052d1a8d21)

**Reply by mehalter** [reply]
https://github.com/AstroNvim/AstroNvim/issues/1013

A good place to check are the issues on the github page. Also the AstroNvim discord is a great place to ask questions :)
